README.md

```diff
@@ -207,6 +207,21 @@ For sync communication we have a community slack with a #containerd channel that
 
 __If you are reporting a security issue, please reach out discreetly at security@containerd.io__.
 
+## Applying License Header to New Files
+
+If you submit a contribution that adds a new file, please add the license
+header. You can do so manually or use the `ltag` tool:
+
+
+```console
+$ go get github.com/kunalkushwaha/ltag
+$ ltag -t ./script/validate/template
+```
+
+The above will add the appropriate licenses to Go, Makefile, Dockerfile files & Bash-scripts. New templates will
+need to be added if other kinds of files are added. Please consult the
+documentation at https://github.com/
+
```

Comment:
wrong url

Reply:
There is no particular page as such for this. It just tells  to understand license header read documentation at github.com

The text is actually copied from [github.com/containerd/btrfs](https://github.com/containerd/btrfs/blob/master/README.md#applying-license-header-to-new-files)

If you feel, its  confusing, I can remove the text `Please consult the documentation at https://github.com/`

Reply:
I thought you meant https://github.com/kunalkushwaha/ltag

Also, I wonder we can move this notation to a file under `docs` directory.

Reply:
currently, `docs` folder containers container.io website files.

Though this can be added to [CONTRIBUTING.md](https://github.com/containerd/containerd/blob/master/CONTRIBUTING.md) also instead of readme.

Reply:
I think we have to specify what documentation we are talking about, because even I can't find anything about adding license headers to files :) The closest thing I found from github.com is: https://help.github.com/articles/licensing-a-repository/   Unless we have something specific we want developers to understand about licensing, I think we might as well leave that sentence off.